src/libraries/System.Runtime/ref/System.Runtime.cs

```diff
@@ -46,6 +46,8 @@ namespace System
     public partial class AccessViolationException : System.SystemException
     {
         public AccessViolationException() { }
+        [System.ObsoleteAttribute("This API supports legacy formatter-based serialization. It should not be called or extended by application code.", DiagnosticId = "SYSLIB0051", UrlFormat = "https://aka.ms/dotnet-warnings/{0}")]
```

Comment:
Instead of using the standard `https://aka.ms/dotnet-warnings/{0}` UrlFormat for all of these obsoletions, I wonder if we should use `https://aka.ms/binaryformatter`.

If we stick with the standard one, it can still be a landing page for that _specific_ diagnostic and then link out to the bigger document. That makes me lean toward leaving it as you have it here, but I wanted to call it out as something to consider.

/cc @gewarren